src/Makefile

```diff
@@ -12,9 +12,9 @@ LINKFLAGS = -I $(CAMLP5D)
 include $(ROOT)/tools/Makefile.common
 
 compilation.ml: always
-	echo "Util.set_compilation_time \"$$(date "+%Y-%m-%d %T %z")\";" > $@
+	echo "Util.set_compilation_time \"$$(date "+%Y-%m-%d %T")\";" > $@
 	echo "Util.set_commit \"$$(git show -s --pretty=format:%h)\";" >> $@
-	echo "Util.set_commit_date \"$$(git show -s --pretty=format:%ci)\";" >> $@
+	echo "Util.set_commit_date \"$$(git show -s --pretty=format:%ci | cut -c 1-16)\";" >> $@
```

Comment:
https://stackoverflow.com/questions/7853332/how-to-change-git-log-date-formats details the poor [git doc](https://github.com/git/git/blob/master/Documentation/date-formats.txt) at least for that `--date` part and shows that we can use the same variables as above, so I propose to use: `date "+%d %B %Y %H:%M"` and `git show -s --pretty=format:%cd --date=format:'%d %B %Y %H:%M'` that looks more like the format of dates used in GeneWeb.  `%c` is an alternative, it uses locale and is nearly the same on my computer (a bit longer with day name and seconds) but could perhaps be more user friendly for everybody. We were asking on chat if date without time wasn't enough, on which case %d %B %Y would be enough